atlp-rwanda/falcons-ec-fe
# #185106765 SASS set up

**What does this PR do?**

- Setup SASS processing on the client

**Description of Task to be completed?**

Install and setup sass pre-processing on the application to enable the use of both .scss and .css files on the client

**How should this be manually tested?**

- clone the project and use the terminal to start the development server by "npm run dev" and "npm run build"
- you can also copy the link in the src comp in the screenshoot space and paste it in the browser to see how it changed the app.

**Any background context you want to provide?**
none

**What are the relevant pivotal tracker/Trello stories?**
Tracker: #185106765

Screenshots (if appropriate)
<img width="1440" alt="Screenshot 2023-05-08 at 16 24 59" src="https://user-images.githubusercontent.com/90443432/237018199-fecfab16-873d-4703-9a9b-bba5aebf8ad0.png">

Questions:
none
